spec/spec_helper.rb

```diff
@@ -1,21 +1,28 @@
 # Make sure specs run with the definitions from test.rb
 ENV['ROBOT_ENVIRONMENT'] = 'test'
 
+require 'simplecov'
 require 'coveralls'
 Coveralls.wear!
 
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
+
 bootfile = File.expand_path(File.dirname(__FILE__) + '/../config/boot')
 require bootfile
 
 require 'pry'
 require 'rspec'
+require 'webmock/rspec'
 
 def setup_work_item(druid)
   @work_item=double("work_item")
   allow(@work_item).to receive_messages(:druid=>druid)
 end
 
-def setup_release_item(druid,obj_type,item_members=nil)
+def setup_release_item(druid,obj_type,item_members,collections)
```

Comment:
Is there a reason why you didn't default item_members and collections in the def params to nil?